packages/@icon-magic/distribute/src/create-icon-template.ts

```diff
@@ -0,0 +1,30 @@
+
+import { Asset } from '@icon-magic/icon-models';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { DOMParser } from 'xmldom';
+
+/**
+ * Saves svg assets as handlebars files
+ * @param assets SVG assets to convert
+ * @param outputPath path to write to
+ */
+export async function createHbs(
+  assets: Asset[],
+  outputPath: string,
+): Promise<void> {
+  for (const asset of assets) {
+    const doc = new DOMParser();
+    // Get contents of the asset, since it's an SVG the content will be in XML format
+    // Content Buffer | string
+    const contents = await asset.getContents();
+    // Parse XML from a string into a DOM Document.
+    const xml = doc.parseFromString(contents as string, 'image/svg+xml');
+    const id = xml.documentElement.getAttributeNode('id');
+    const iconName = id ? id.value : '';
+    fs.writeFile(path.join(outputPath, `${iconName}.hbs`), xml, (err) => {
```

Comment:
I might be missing some typescript magic here, but doesn't this need something to handle the asynchrony? Seems like this could leak and fail outside of your try/catch block in `distributeSvg`